src/dev/build/tasks/os_packages/docker_generator/templates/base/Dockerfile

```diff
@@ -231,7 +231,7 @@ ENTRYPOINT ["/bin/tini", "--"]
 CMD ["/app/kibana.sh"]
 # Generate a stub command that will be overwritten at runtime
 RUN mkdir /app && \
-    /usr/bin/echo -e '#!/bin/bash\nexec /usr/local/bin/kibana-docker' > /app/kibana.sh && \
+    /bin/echo -e '#!/bin/bash\nexec /usr/local/bin/kibana-docker' > /app/kibana.sh && \
```

Comment:
`/bin/echo` is needed to support the `-e` flag, otherwise the sh built in is used.  `/bin` is symlinked to `/usr/bin` on ubi and ubuntu.  on chainguard it is not.